docs/source/docformat.rst

```diff
@@ -4,35 +4,276 @@ Documentation Formats
+``pydoctor`` needs the following packages to offer *restructuredtext* support::
+
+   $ pip install -U docutils Pygments
+
+Read the `RST syntax reference <https://docutils.sourceforge.io/docs/user/rst/quickref.html>`_.
+
+Fields
+^^^^^^
+
+As a reminder, here are some of the supported *restructuredtext* fields:
+
+    - ``:cvar foo:``
+    - ``:ivar foo:``
+    - ``:var foo:``
+    - ``:param bar:`` (synonym: ``:arg bar:``)
+    - ``:type bar: str``
+    - ``:return:``
+    - ``:rtype: list``
+    - ``:except ValueError:``
+
+Alternatively, fields can be passed with this syntax::
+
+    :Parameters:
+        size
+            The size of the fox (in meters)
+        weight : float
+            The weight of the fox (in stones)
+        age : int
+            The age of the fox (in years)
+
+Directives
+^^^^^^^^^^
+
+.. list-table:: Summary of supported and unsupported directives, with links to appropritate reference. 
+   :header-rows: 1
+   
+   * - Directive
+     - Reference
+     - Support
+
+   * - ``.. include::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment>`_
+     - Yes
+
+   * - ``.. contents::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#table-of-contents>`_
+     - Yes
+
+   * - ``.. image::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#image>`_
+     - Yes
+       
+   * - ``.. |time| date:: %H:%M``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#date>`_
+     - Yes
+
+   * - ``.. figure::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure>`_
+     - Yes
+
+   * - ``.. |T| replace:: term``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#replacement-text>`_
+     - Yes
+ 
+   * - ``.. unicode::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#unicode-character-codes>`_
+     - Yes
+ 
+   * - ``.. raw::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#raw-data-pass-through>`_
+     - Yes
+  
+   * - ``.. class::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#class>`_
+     - eh
+  
+   * - ``.. role::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#custom-interpreted-text-roles>`_
+     - Yes
+  
+   * - ``.. default-role::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#setting-the-default-interpreted-text-role>`_
+     - Yes
+    
+   * - ``.. line-block::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#line-block>`_
+     - eh
+
+   * - ``.. code::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#code>`_
+     - Yes (syntax highlight ignored). Use ``.. python::``. 
```

Comment:
I think the problem might be that the CSS is missing, similar to what happened for the `python` directive before.

Pygments can generate the required CSS, but if nothing is requesting this CSS from Pygments and writing it to file, the browser won't have access to it.